web/gds-user-ui/src/modules/dashboard/member/utils.ts

```diff
@@ -49,3 +49,26 @@ export const downloadMembers2CVS = (member: any) => {
   const memberCsv = convertToCVS(m, memberTableHeader as ITableHeader[]);
   downloadCSV(memberCsv, 'members');
 };
+
+export const getVaspStatus = (status: string) => {
```

Comment:
The statuses come from here: https://github.com/trisacrypto/trisa/blob/main/pkg/trisa/gds/models/v1beta1/models.pb.go#L82C1-L82C1

Reply:
Would you mind updating L43-48 with this
```js
  const m = member.map((item: any) => {
    return {
      ...item,
      status : getVaspStatus(item.status)
      registered_directory: getVapsNetwork(item.registered_directory)
    };
  });

```

Reply:
I've merged in the main branch so now this should be included.